source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/product-definition-select/display.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { Dropdown } from 'semantic-ui-react';
+
+import { attributesFor, ProductDefinitionResource } from '@data';
+import { i18nProps } from '@lib/i18n';
+import { filterForValidAttributes } from '@lib/dom';
+
+interface IOwnProps {
+  className?: string;
+  productDefinitions: ProductDefinitionResource[];
+  onChange: (value: string) => void;
+  defaultValue: string;
+}
+
+export type IProps =
+& IOwnProps
+& i18nProps;
+
+export class Display extends React.Component<IProps> {
+  onChange = (e, dropdownEvent) => {
+    e.preventDefault();
+
+    this.props.onChange(dropdownEvent.value);
+  }
+
+  render() {
+    const { productDefinitions, defaultValue, t, ...otherProps } = this.props;
+
+    const options = [{ text: t('productDefinitions.all'), value: 'all' }].concat(
+      productDefinitions.map(pd => ({
+        text: attributesFor(pd).name || '',
+        value: attributesFor(pd).name
```

Comment:
shouldn't be the id instead of name?

Reply:
I was thinking about maybe making the value customizable. but it's like this because the backend query is a LIKE query -- probably doesn't need to be